xml/rmt_mirroring.xml

```diff
@@ -190,19 +200,27 @@ Jun 22 04:22:34 d31 systemd[1]: Started RMT Mirror timer.</screen>
     Example:
    </para>
 <screen>&prompt.sudo;<command>rmt-cli repo list --all</command>
```

Comment:
```suggestion
<screen>&prompt.sudo;<command>rmt-cli repos list --all</command>
```

`repo` is not really a subcommand of rmt-cli. It only works because Thor (our cli builder library) is happy as long as the command you enter is an unambiguous substring of an existing one. So even `rep li --all` would work ... but please don't document this :wink: 

The correct subcommand name is `repos`.